### PR TITLE
Add: Africa & Australia

### PR DESCRIPTION
**Pull request by blckbx**
This PR adds Africa (ZA - Johannesburg) and Australia (Sydney) as VPNs to TunnelSat's frontend and backend.

### CODE REVIEW

File: frontend/client/src/components/WorldMap.js [hunk 1]
```diff
@@ -46,10 +46,40 @@ const WorldMap = (props) => {
...
+        <OverlayTrigger
+          trigger="hover"
+          key="top"
+          placement="top"
+          overlay={
+            <Popover id="popover-basic" className="customPopover">
+              <Popover.Title as="h3">Africa</Popover.Title>
+              <Popover.Content>
+                <IoLocation color="#ffc700" size={20} />{" "}
+                <strong>Server Location: </strong>Cape Town
```
Comment (TrezorHannes) [inline frontend/client/src/components/WorldMap.js]: Shouldn't this be "Johannesburg"

File: frontend/server/server.js [hunk 2]
```diff
@@ -401,6 +401,7 @@ io.on("connection", (socket) => {
       { domain: "us1.tunnelsats.com", country: "na" },
       { domain: "sg1.tunnelsats.com", country: "as" },
       { domain: "br1.tunnelsats.com", country: "sa" },
+      { domain: "ct1.tunnelsats.com", country: "af" },
```
Comment (TrezorHannes) [inline frontend/server/server.js]: ` { domain: "ct1.tunnelsats.com", country: "af" },` - verifying whether we have the right sub-domain settings for za1 here

**Review by TrezorHannes — APPROVED** [review]
LGTM

**Review by ziggie1984 — APPROVED** [review]
LGTM

**Pull request merged**